community/pmc-tool/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
let's remove everything related to Django. Let's focus only on downloading data and storing it in SQLite.